spec/index.bs

```diff
@@ -128,20 +128,24 @@ The <dfn>contact picker task source</dfn> is a [=task source=].
 A <dfn>user contact</dfn> consists of:
 <div dfn-for="user contact">
 
-* <dfn>names</dfn>, a [=list=] of {{DOMString}}s, initially empty, each [=list/item=] representing
-  a unique name corresponding to the user.
-* <dfn>emails</dfn>, a [=list=] of {{DOMString}}s, initially empty, each [=list/item=] representing
-  a unique [=valid e-mail address=] of the user.
-* <dfn>numbers</dfn>, a [=list=] of {{DOMString}}s, initially empty, each [=list/item=]
-  representing a unique phone number of the user.
-* <dfn>addresses</dfn>, a [=list=] of {{ContactAddress}}es, initially empty, each [=list/item=]
-  representing a unique [=physical address=] of the user.
+* <dfn>names</dfn>, a [=list=] of {{DOMString}}s, each [=list/item=] representing a unique name
+  corresponding to the user.
+* <dfn>emails</dfn>, a [=list=] of {{DOMString}}s, each [=list/item=] representing a unique
+  [=valid e-mail address=] of the user.
+* <dfn>numbers</dfn>, a [=list=] of {{DOMString}}s, each [=list/item=] representing a unique phone
+  number of the user.
+* <dfn>addresses</dfn>, a [=list=] of {{ContactAddress}}es, each [=list/item=] representing a
+  unique [=physical address=] of the user.
+* <dfn>icons</dfn>, a [=list=] of {{Blob}}s, each [=list/item=] representing a unique image of the
+  user.
+    
+    NOTE: An icon {{Blob}} is image decodable and the {{Blob/type}} is an [=image mime type=].
```

Comment:
What does _image decodable_ mean?

Reply:
This is how the manifest spec defined a blob that can be interpreted as an image, but it seems to have been removed now (even then it didn't link to anything). I'll remove this as well, the mime type restriction should be enough.

Reply:
I think it's useful to explain what to do with the blob, even if done as an example. Specs with more verbosity on use-cases and/or examples is something I'd love to see more of.

Reply:
I added an extensive example showing how to use a blob to:
- Display an image
- Create a Bitmap
- Upload to endpoint